packages/manager/apps/web/client/app/hosting/dashboard/hosting.service.js

```diff
@@ -444,6 +446,73 @@ import union from 'lodash/union';
         );
       }
 
+      /**
+       * Get available plans to detach a hosting
+       * @param {string} serviceId
+       */
+      getAvailablePlanDetach(serviceId) {
+        return this.$http
+          .get(`/services/${serviceId}/detach`)
+          .then(({ data }) => data);
+      }
+
+      /**
+       * Get estimation for detach hosting operation
+       * @param {string} serviceId
+       * @param {string} planCode
+       */
+      simulateEstimate(serviceId, planCode) {
+        return this.$http
```

Comment:
You can retrieve all these methods here: https://github.com/ovh/manager/blob/master/packages/manager/modules/product-offers/src/services/product-offers-action.service.js

Reply:
Thanks for your review. Indeed it makes sense to use modules if functionality is available. I'll update PR with this modif.

Reply:
PR is updated